### PR TITLE
MAIN - adds storybook-addon-a11y

### PR DESCRIPTION
**Pull request by paschalidi**
Hello @Weetbix and @lanwen!! 

Since lately I have the feeling we want to take care of accessibility its a good start to have [this little addon](https://github.com/storybooks/storybook/blob/next/addons/a11y/README.md) that with minimum effort on our side can provide us with some accessibility tests on our vivy components!! 

What a time to be alive! :D 

### How does this look like you ask?
image

### CODE REVIEW

**Comment by paschalidi** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Missing reviewer - <i>Looks like this pull request is missing a reviewer. That's OK as long as this PR is still a 'work in progress', if that is the case please add 'WIP' in the PR title or corresponding label.</i></td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>✅  Get a 🏆 for adding description on this PR. - <i>Celebrate on every step!</i></td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b756d22d8422cb5705d28c9c1a5c792712fd6d75
</p>

**Comment by Weetbix** [comment]
@dependabot-bot rebase

**Comment by paschalidi** [comment]
right, will take care of this!

**Comment by paschalidi** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Missing reviewer - <i>Looks like this pull request is missing a reviewer. That's OK as long as this PR is still a 'work in progress', if that is the case please add 'WIP' in the PR title or corresponding label.</i></td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>✅  Get a 🏆 for adding description on this PR. - <i>Celebrate on every step!</i></td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against dcea586850131977489cc53ad0330f2daf25af10
</p>

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "test:watch": "npm run test -- --watchAll"
   },
   "dependencies": {
-    "@storybook/theming": "^5.0.6",
+    "@storybook/theming": "^5.1.9",
```
Comment (paschalidi) [inline package.json]: maybe we want to have this as dev dependency, I dont see any reason that this is up here.

Comment (Weetbix) [reply package.json]: 👍

**Pull request closed without merging**